config.py

```diff
@@ -0,0 +1,24 @@
+"""Flask configuration."""
+import os
+from os import environ, path
+from dotenv import load_dotenv
+
+basedir = path.abspath(path.dirname(__file__))
+load_dotenv(path.join(basedir, '.env'))
+
+
+class Config:
+    TESTING = False
+    DEBUG = False
+
+
+class ProdConfig(Config):
+    FLASK_ENV = 'production'
+    PATH_FOR_INITIAL_DATA = os.environ.get("PATH_FOR_INITIAL_DATA") or path.join(basedir, 'restaurants.json')
+
+
+class DevConfig(Config):
+    FLASK_ENV = 'development'
```

Comment:
probably `FLASK_ENV` should not be defined as `Config` property. `FLASK_ENV` is an environment variable that actually defines which config to use

Reply:
yet `FLASK_ENV` as property here does not hurt too, I guess

Reply:
fixed